demo/src/style.css

```diff
@@ -69,11 +69,14 @@ summary {
 
 
 .imagePanel {
-
 	display: grid;
 	place-items: center;
 }
 
+.imagePanel img {
+	object-fit: cover;
```

Comment:
@ascorbic I added this so that the `img` behavior matches the CDN ones

previously if you typed in a new value for width/height (esp one that the CDN needs to now generate) it would show a distorted image (stretch the image, which I think there is pretty much no case where people would want to do this in production - aka the default `object-fit`) and then it snaps into place when the CDN is updated

since all CDNs by default use `cover`, I changed the img behavior accordingly so when you change width/height it won't look distorted while the new one loads. in most cases you just see it be pixelated for a sec, and then it snaps to being nice and clear

felt more right/intuitive, but happy to remove if intentionally undesired

Reply:
I was going to change that myself, so thanks!